account_payment_credit_card/models/account_move.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 Open Source Integrators <http://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def post(self, invoice=False):
```

Comment:
```suggestion
    def post(self):
```
In Odoo13 ```def post``` not require "invoice"